src/test/kotlin/racingcar/RacingTest.kt

```diff
@@ -33,15 +33,15 @@ internal class RacingTest {
 
         racing.start(1, strategy)
 
-        assertAll("", {
-            assertThat(racing.result[0].cars[0].position).isEqualTo(1)
-            assertThat(racing.result[0].cars[1].position).isEqualTo(1)
+        assertAll("자동차 포지션 테스트", {
+            assertThat(racing.history[0].cars[0].position).isEqualTo(1)
+            assertThat(racing.history[0].cars[1].position).isEqualTo(1)
         })
     }
 
     @Test
     fun `잘못된 결과 값 테스트`() {
-        val racing = Racing(Cars(listOf(Car(), Car())))
+        val racing = Racing(Cars(listOf(Car("a"), Car("b"))))
 
         val strategy = object : MoveStrategy {
             override fun isMove(): Boolean {
```

Comment:
```suggestion
        val strategy = MoveStrategy { true }
```
전에 `@FunctionalInterface`를 이용하는 부분에 대한 피드백에서 SAM에 대해 언급하셔서 알아보았습니다!

MoveStartegy interface를 fun interface로 만들면 SAM을 사용하게 되면서, 코드가 더 간결해지더군요!

<img width="260" alt="스크린샷 2022-05-17 오전 7 30 35" src="https://user-images.githubusercontent.com/50702723/168692998-765fb1d0-14bd-4469-a554-cc1ae9f645a8.png">

참고
- [Functional (SAM) inteface](https://kotlinlang.org/docs/fun-interfaces.html)

Reply:
아하 fun 이 붙었어야 하는 군요!!! 👍